core/src/com/group66/game/screens/SettingsScreen.java

```diff
@@ -2,89 +2,91 @@
 package com.group66.game.screens;
 
 import com.badlogic.gdx.Gdx;
-import com.badlogic.gdx.Screen;
-import com.badlogic.gdx.graphics.Color;
 import com.badlogic.gdx.graphics.GL20;
-import com.badlogic.gdx.graphics.Pixmap;
-import com.badlogic.gdx.graphics.Pixmap.Format;
-import com.badlogic.gdx.graphics.Texture;
-import com.badlogic.gdx.graphics.g2d.BitmapFont;
 import com.badlogic.gdx.scenes.scene2d.Actor;
 import com.badlogic.gdx.scenes.scene2d.Stage;
-import com.badlogic.gdx.scenes.scene2d.ui.Skin;
 import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
-import com.badlogic.gdx.scenes.scene2d.ui.TextButton.TextButtonStyle;
 import com.badlogic.gdx.scenes.scene2d.utils.ChangeListener;
 import com.group66.game.BustaMove;
-import com.group66.game.helpers.AssetLoader;
 import com.group66.game.helpers.DifficultyManager;
 import com.group66.game.logging.MessageType;
 import com.group66.game.settings.Config;
 
 /**
  * A Class for the SettingsScreen of the game.
  */
-public class SettingsScreen implements Screen {
+public class SettingsScreen extends AbstractMenuScreen {
 
-    private Stage stage;
-    private Skin skin;
```

Comment:
Turns out Shop screen is also being disposed off correctly.
The YouWinScreenCareer isn't, so I assume all children of AbstractMenuScreen dispose of stuff correctly. Can you take a look and gimme your opinion?